docs/specs/clients/notify/authentication.md

```diff
@@ -118,8 +118,243 @@ A non-ideal way to avoid the race condition is for the sender to set the message
 
 ## wc_notifyDelete response
 
-- act - `notify_delete_response``
+- act - `notify_delete_response`
 - iss - did:key of dapp authentication key
 - aud - did:key of client identity key
 - app - did:web of app domain that this request is associated with 
   - Example: `did:web:app.example.com`
+
+## `wc_notifyGetNotifications`
+
+Paginated list of notifications with the most recent first. Unread notifications may be dispersed throughout the list and on subsequent pages.
+
+### Request
+
+- act - `notify_get_notifications`
+- iss - did:key of client identity key
+- ksu - key server for identity key verification
+- aud - did:key of dapp authentication key
+- app - did:web of app domain that this request is associated with 
+  - Example: `did:web:app.example.com`
+- lmt - the max number of notifications to return. Default value is 10, maximum value is 50.
```

Comment:
What is the behaviour of the server when limit is reached. Capping or error response?

Reply:
Error response

Reply:
Did you do any math to come up with limit or is maximum of 50 messages a magic number?

Reply:
Also not sure if defaulting is necessary

Reply:
> Did you do any math to come up with limit or is maximum of 50 messages a magic number?

We need some kind of limit to protect the system, and I thought 50 was a very liberal limit for displaying a list of notifications on a UI.

Reply:
> Also not sure if defaulting is necessary

Good point, I'd be open to removing this.